src/core/thermostat.cpp

```diff
@@ -144,3 +145,124 @@ void philox_counter_increment() {
     thermalized_bond.rng_increment();
   }
 }
+
+#ifndef PARTICLE_ANISOTROPY
+void mpi_set_brownian_gamma_local(double gamma) {
+  brownian.gamma = GammaType{gamma};
+}
+
+void mpi_set_brownian_gamma_rot_local(double gamma) {
+  brownian.gamma_rotation = GammaType{gamma};
+}
```

Comment:
CI didn't pass the coverage checks because a few callbacks like this one are never executed with the config files available in CI. This can be solved by removing duplicated callbacks: e6043ef6d3a123a1266a96bed133bf5e28302813

Reply:
Fixed.